src/pybind/mgr/balancer/module.py

```diff
@@ -72,7 +70,8 @@ def final_state(self):
         self.inc.set_osd_reweights(self.osd_weights)
         self.inc.set_crush_compat_weight_set_weights(self.compat_ws)
         return MappingState(self.initial.osdmap.apply_incremental(self.inc),
-                            self.initial.pg_dump,
+                            self.initial.raw_pg_stats,
```

Comment:
```
2020-01-15T14:03:58.128+0000 7fcac36a0700 -1 balancer.serve:
2020-01-15T14:03:58.128+0000 7fcac36a0700 -1 AttributeError: 'MappingState' object has no attribute 'raw_pg_stats'
```

raw_pg_stats is not assigned to `self.raw_pg_stats` in the constructor of `MappingState`.